Encounter Me.Api/Models/AppDbContext.cs

```diff
@@ -25,7 +25,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<UserData>().HasData(new UserData
             {
-                Id = 1,
+                Id = Guid.NewGuid(),
                 UserName = "obuolys",
                 FirstName = "Dominykas",
                 LastName = "Zagreckas",
```

Comment:
Is this dummy data still needed?  It doesn't seem necessary anymore as we can add new users for testing purposes via sign up form. Also, it seems that code for it's creation is duplicated in at least two places.

Reply:
I removed the dummy data.